spec/features/user_visits_registration_page_spec.rb

```diff
@@ -16,7 +16,7 @@
     fill_in("Password", with: user.password_digest)
     click_button "Register"
 
-    expect(page).to have_css("p", text: "Welcome Dummy User!")
+    expect(page).to have_css("h3", text: "Sign In")
```

Comment:
You don't want to just make the tests pass, you need to test for something relevant. In this scenario you need to test for something that's only present on the page AFTER a successful sign up (in this case the user profile page)

If you tests are passing by testing for "Sign In" that means that it wasn't a successful sign up!